Comparison shopping engine/Scrapers/Selenium/AutoaibeScraper.cs

```diff
@@ -79,8 +79,7 @@ protected override bool ShouldScrapeIf(IWebElement product)
 
         protected override (string, string, string, string) GetInfo(IWebElement product)
```

Comment:
Ar nereiktų viską sudėti į try catch blokus, kad netyčia atsitikus tokiam atvėjui, jog nerado to produkto informacijos nemestų exceptiono? Ar kažkaip kitaip su tuo susidoroti.

Reply:
galbūt būtų paprasciau abstract klaseje pridėti dar viena if'a po sito metodo iskvietimo ir patikrinti ar kazkuris laukas yra null, jeigu taip, tuomet ta produkta praleistume, ka manai?Tuomet nereikės eiti per visa scraperiu klases kad pridėti try catch. Lauksiu atsakymo ir jeigu tinka pridėsiu tikrinimą. Arba galima būtų ir abstract klasej dėti trycatch, bet nezinau ar taip suveiktu. 
🤔

Reply:
Manau pavyktų su try catch'u abstrakčioje klasėje